tests/unit_10.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1 - Verify login with valid credentials', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+  await page.locator('[data-test="email"]').fill('customer@practicesoftwaretesting.com');
+  await page.locator('[data-test="password"]').fill('welcome01');
```

Comment:
I'd use the getByTestId method to find an element. But you must do this before https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute